libraries/AR_WPNav/AR_WPNav.h

```diff
@@ -114,8 +114,6 @@ class AR_WPNav {
     // adjust speed to ensure it does not fall below value held in SPEED_MIN
     void apply_speed_min(float &desired_speed);
 
-private:
```

Comment:
no need to remove this "private" I think.

Reply:
fixed